tasks/main.yml

```diff
@@ -98,16 +58,12 @@
     groups: "{{ osbs_admin_groups }}"
 
   - name: osbs-custom-build-readwrite
-    role: osbs-custom-build
-    yaml_version: v1
-    role_namespace: "{{ osbs_namespace }}"
+    role: system:build-strategy-custom
```

Comment:
Doesn't using clusterroles require elevated privileges?

Reply:
The role is defined in a cluster, yes, but the binding happens per namespace (as `RoleBinding` is default, unlike `pruner) - admin would be able to run custom builds on current namespace only, not the whole cluster.

I'm not sure if that would work on OSD right away, but we could ask for a permission to run custom builds, as the changes would be limited to the namespace only.